src/Datadog.Trace/Configuration/TracerSettings.cs

```diff
@@ -285,5 +283,11 @@ internal bool IsIntegrationEnabled(string name)
             var analyticsEnabled = integrationSettings.AnalyticsEnabled ?? (enabledWithGlobalSetting && AnalyticsEnabled);
             return analyticsEnabled ? integrationSettings.AnalyticsSampleRate : (double?)null;
         }
+
+        private bool IntegrationIsNotExplicitlyDisabled(string name)
```

Comment:
I think a new method is overkill when a comment inside `IsIntegrationEnabled()` would have been enough to clarify this. If we decide to go this route, however, I think the `Not` in the method name is confusing. The integration is enabled if it is _not not_ disabled? I would reverse the logic to something like:
```csharp
private bool IntegrationIsDisabled(string name)
{
    return Integrations[name].Enabled == false ||
           DisabledIntegrationNames.Contains(name);
}
```
and change `IsIntegrationEnabled()` to check for `TraceEnabled && !IntegrationIsDisabled(name)`.

Reply:
Agree -- good idea.